files/twix.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "twix.js",
+  "repo": "icambron/twix.js",
+  "files": {
+    "basePath": "bin/"
```

Comment:
Add an include parameter  with `"*.min.js"` to make sure we host only the minified files